### PR TITLE
Add more comprehensive docs

### PR DESCRIPTION
**Pull request by kubkon**
Changes include:
* more comprehensive README
* simple CONTRIBUTING guide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,176 @@
...
+int main(int argc, char** argv) {
+  if (argc != 2) {
+      printf("You need to specify your name!\n");
+      return -1;
+  }
+  
+  FILE* f_in = fopen("in.txt", "r");
+  FILE* f_out = fopen("out.txt", "w");
+  
+  fprintf(f_out, "Hello world!\n");
```
Comment (mplebanski) [inline README.md]: This "hello world" print seems redundant to me.

Comment (kubkon) [reply README.md]: Yeah, that's fair enough!

Comment (kubkon) [reply README.md]: Removed :+1:

File: README.md [hunk 2]
```diff
@@ -8,13 +8,173 @@
...
+  - [License](#license)
+
+## Quick start guide
+This guide assumes you have successfully built the `wasm-sandbox` binary; for build instructions, see section
+[Build instructions](#build-instructions) below.
+
+### 1. Create simple C program
+Let us create a simple `hello world` style C program which will read
+in some text from `in.txt` text file, read your name from the command
+line, and save the resultant text blob in `out.txt`.
```
Comment (mplebanski) [inline README.md]: blob stands for binary large object -> text binary large object does not really make sense to me here

Comment (kubkon) [reply README.md]: Good point!

Comment (kubkon) [reply README.md]: Removed :+1:

**Pull request merged**